GCMessengerSDKSample/app/src/main/java/com/genesys/gcmessengersdksample/data/defs/ChatType.java

```diff
@@ -0,0 +1,17 @@
+package com.genesys.gcmessengersdksample.data.defs;
+
+
+import androidx.annotation.StringDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.SOURCE)
+@StringDef({ChatType.Live, ChatType.Bot, ChatType.Messenger, ChatType.ChatSelection, ChatType.ContinueLast})
+public @interface ChatType {
+    String ChatSelection = "Chat selection";
+    String Live = "Live Chat";
```

Comment:
I think the live chat and bot chat can be remove from this sample.
We will preserve the chat forms for this types but ww won't show it exists